src/feed_handler.py

```diff
@@ -83,6 +92,27 @@ def _update_height(self, clob_pair_id: int, new_block_height: int):
                 f"Block height decreased from "
                 f"{self.heights[clob_pair_id]} to {new_block_height}"
             )
+
+    def _handle_subaccounts(self, update: subaccounts.StreamSubaccountUpdate):
+        """
+        Handle the StreamSubaccountUpdate message, updating the local subaccount state.
+        """
+        parsed_subaccount = subaccounts.parse_subaccounts(update)
+        subaccount_id = parsed_subaccount.subaccount_id
+
+        if update.snapshot:
```

Comment:
Can we follow [this](https://github.com/dydxprotocol/grpc-stream-client/blob/c06bd30dd2c1550ef621743c812d405670e6384d/src/feed_handler.py#L139-L141) pattern?

No need to replace everything if there is a snapshot update, just ignore it

Reply:
There will be a snapshot per subaccount id, and this will be different from the fills. I think it's easier to do it the existing way.